### ISSUE TITLE
E15/E1 - Two different listing views share same title

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#146]
Both listing pages for Upcoming Events and Past Events are titled "Events Calendar". The title of the page does not distinguish which page is which, failing accessibility criterion 2.4.2.

**Comment by mmafe** [comment]
New titles on calendar:
- Upcoming Events
- This Week
- Monthly Calendar
- Past Events

**Comment by jpinsent** [comment]
Tested on the OAC stage site and AODA Demo and the issue has been resolved.
